### DISCUSSION TITLE
The 100 years old Einstein - Selety correspondence

### DISCUSSION THREAD

**Discussion by HanDeBruijn** [a-cosmology-group/acg#132] [Cosmology Models]
From __[Fractal Approach to Large-Scale Galaxy Distribution](https://arxiv.org/abs/astro-ph/0505185)__ we copy.

### 6.3.1 Einstein's Cosmological Principle

In the current cosmological thinking the contents of the Cosmological Principle are rather deep and complicated.
We point out three distinct statements which are usually regarded as a starting point for constructing cosmological models.

CP1 _Physical laws the same in all space and time_
CP2 _Fundamental physical constants are true constants_
CP3 _Particular physical properties, including measuring standards, are the same in all space and time_

### 4.1.1 Einstein - Selety correspondence

Selety (1922) argued that it is possible to construct hierarchical worlds which fulfil simultaneously the following conditions:

• _infinite space_
• _infinite total mass_
• _mass filling space so that locally there is everywhere a finite density_
• _zero average density of the mass in the whole world_
• _non-existence of a unique middle point or middle region of the world_

I'm curious what ACG members think about the above 100 years old _theoretical principles_. Outdated or not?

**Comment by ftessier** [comment]
In regards to 6.3.1, to me the principles are outdated to the extent that the nature of space and time pose ontological questions that ought to be investigated. As written, these core principles present space and time as fundamental concepts, axiomatic "things", instead of phenomena requiring explanation.

In terms of 4.1.1, I love fractal geometries, but I am not comfortable with the concept of infinity, beyond its mathematical usefulness, so no comment...

**Reply by HanDeBruijn** [reply]
>these core principles present space and time as fundamental concepts, axiomatic "things", instead of phenomena requiring explanation.

I agree with you concerning "time". I disagree with you concerning "space". Our __geometry__ of space is a _careful abstraction_ from physical reality, followed by an _idealization_ in the form of _axiomatic "things"_, yes.
But I would describe it as __proper mathematics__ as it has been done by [Euclid](https://en.wikipedia.org/wiki/Euclid), IMHO not at all comparable with nowadays work. Therefore I am very much against "explaining" space in other ways than by Euclidean geometry.
Especially worth mentioning in this context is [General Relativity, blessing or curse?](https://github.com/orgs/a-cosmology-group/discussions/78) There we find that even [elementary algebra would collapse](https://math.stackexchange.com/questions/1951863/non-euclidean-geometrical-algebra-for-real-times-real) without Euclidean geometry.

It is funny that with the advent of Computer Graphics they had to invent trivial things as how to make finite mappings of an infinite planar geometry. One of the issues in the old days has been the [Clipping](https://www.cc.gatech.edu/grads/h/Hao-wei.Hsieh/Haowei.Hsieh/mm.html) problem. An example is found [here](https://math.stackexchange.com/questions/901259/the-area-of-the-region-x-ay-le-c-for-0-le-x-le-1-and-0-le-y-le-1). Another issue has been how to draw a straight line, if pixels is the only thing you have. Surprisingly, the [Euclidean algorithm](https://en.wikipedia.org/wiki/Euclidean_algorithm) is one of the methods ready to accomplish the task.

Understanding "time" is quite [another matter](https://github.com/orgs/a-cosmology-group/discussions/63#discussioncomment-6255770).

**Reply by ftessier** [reply]
Euclid's geometry is certainly fantastic! However, my preoccupation is this: does space exist independently of physical events? Or is space always the manifestation of a relation between events? Apparently, the notion of an independent space (and time) "scaffolding" is a fairly recent one starting with Newton. Going back to Antiquity, and famously up to Leibniz and Kant, space was not conceived as a background stage where "things happen".

**Reply by HanDeBruijn** [reply]
>does space exist independently of physical events?

I think the answer is definitely __no__. An argument is found in the thread [Towards a Radical Change in Fundamental Physics](https://github.com/orgs/a-cosmology-group/discussions/126#discussioncomment-6762408), especially the theorems by Emmy Noether. It has been proved that conservation laws, in physics, are intimately related to _invariance_ with respect to space and time.
Further support for this idea is found in Wikipedia's [QM operators](https://en.wikipedia.org/wiki/Operator_(physics)#Table_of_QM_operators) and Heisenberg's [Uncertainty principle](https://en.wikipedia.org/wiki/Uncertainty_principle). Physical quantities in QM like momentum and energy are coupled to differentials in space and time via Planck's constant. For those who are not averse to detailed math: the above is [linked to Lie groups](http://www.alternatievewiskunde.nl/MijnBoek/Lie.htm).
__Sad remark.__ The original work by Sophus Lie has been very much [warped](https://en.wikipedia.org/wiki/Lie_group) by the concept of manifolds, so much typical for General Relativity.

**Reply by ftessier** [reply]
To clarify, if I understand your arguments correctly, you mean **yes**, as in _space does exist independently of physical events?_ Noether's theorem, relativity, QM, etc., in fact practically (but not quite) all physics today rests on $\large x$ and $\large t$ variables which are _a priori_ notions. I hold that they ought to emerge from more fundamental physical concepts.

**Reply by HanDeBruijn** [reply]
>all physics today rests on $x$ and $t$ variables which are a priori notions. I hold that they ought to emerge from more fundamental physical concepts.

Admittedly, the examples I have given so far may be better characterized as belonging to _feedback_ rather than _emergence_. However, actually, **space and time _have_ been emergent** from physics, many centuries ago. To be precise: emergent from mechanics, from _human activity_, from "manual arts", hence **from technology**. Here comes a quote from THE AUTHOR'S PREFACE in Newton's Principia, confirming this. Humans _are_ part of nature anyway and we don't have to repeat that part of our history every time again.

>The ancients considered mechanics in a twofold respect; as rational, which proceeds accurately by demonstration; and practical. To practical mechanics all the manual arts belong, from which mechanics took its name. But as artificers do not work with perfect accuracy, it comes to pass that mechanics is so distinguished from geometry, that what is perfectly accurate is called geometrical, what is less so, is called mechanical. But the errors are not in the art, but in the artificers. He that works with less accuracy is an imperfect mechanic; and if any could work with perfect accuracy, he would be the most perfect mechanic of all; _for the description of right lines and circles, upon which geometry is founded, belongs to mechanics_. Geometry does not teach us to draw these lines, but requires them to be drawn; for it requires that the learner should first be taught to describe these accurately, before he enters upon geometry; then it shows how by these operations problems may be solved. To describe right lines and circles are problems, but not geometrical problems. The solution of these problems is required from mechanics; and by geometry the use of them, when so solved, is shown; and it is the glory of geometry that from those few principles, brought from without, it is able to produce so manythings. Therefore _geometry is founded in mechanical practice_, and is nothing but that part of universal mechanics which accurately proposes and demonstrates _the art of measuring_. But since the manual arts are chiefly conversant in the moving of bodies, it comes to pass that geometry is commonly referred to their magnitudes, and mechanics to their motion.

**Reply by Francois-Zinserling** [reply]
@HanDeBruijn @ftessier if space and time are emergent, does it mean velocity is fundamental. The fact that we see 
$velocity = distance/time$ 
does not make it emergent from space and time. It may well be the other way round.

May it then be that the true fundamental is the speed of light, and space and time are emergent from that?

**Reply by HanDeBruijn** [reply]
>May it then be that the true fundamental is the speed of light, and space and time are emergent from that?

The following is a quote from the book [Relativity Reexamined](http://www.alternatievewiskunde.nl/einstein/relativity-reexamined_compress.pdf) by [L&eacute;on Brillouin](https://en.wikipedia.org/wiki/L%C3%A9on_Brillouin).
>We witnessed the invention of atomic clocks of incredible accuracy, whose physical properties differ very much from the clocks Einstein imagined. [ .. ] Let us mention here a real difficulty resulting from internationally adopted definitions. The unit of length is based on the wavelength of a spectral line of krypton-86 under carefully specified conditions with accuracy $10^8$. Hence, the same physical phenomenon, a spectral line, is used for two different definitions: length and time, and the velocity $c_0$ of light remains undefined and looks arbitrary. It should be stated, once and for all, whether _a spectral line should be used to define a frequency OR a wavelength, **but not both**_!

Indeed, according to [Wikipedia](https://en.wikipedia.org/wiki/Second#Based_on_caesium_microwave_atomic_clock): "in 1967 the _Thirteenth General Conference on Weights and Measures_ defined the SI second of atomic time". Therefore, whatever right or wrong, _time_ period $T$ (= 1/frequency) and (wave)_length_ $\lambda$ are nowadays related to each other via the speed of light: $\lambda = c_0.T$ . Where it is assumed that $c_0$ is a constant of nature. As Google says:

>speed of light, speed at which light waves propagate through different materials. In particular, the value for the speed of light in a vacuum is **now defined as** exactly 299,792,458 metres per second.

With other words: _length is emergent from time_ via the speed of light. Not because it's a Law of Nature, but because we have simply *defined* it that way; it's a feature of our _technology_. But we have a problem: what if $c_0$ by some clever experiment turns out **not** to be constant ..

**Reply by budrap00** [reply]
The speed of light has never been accurately measured to within a meter. This is despite the fatuous LIGO claim to be able to measure distances to within 10^-20 meters using a classical interferometer composed entirely of 10^-12 meter atoms vibrating chaotically over an order of magnitude, and employing a laser with a 10^-7 meter wavelength. 

According to Einstein's General Relativity the SOL is not constant - it varies with position in a gravitational field. The so-called Einstein Equivalence Principle, which Einstein had nothing to do with, is the only basis for the spurious claim that the speed of light is a universal constant. The EEP is just a lazy conjecture of mathematical convenience adopted mid-20th century with no empirical or even logical justification. 

When observations failed to find a constant SOL, the problem was solved by assigning it, by definition, a constant value. Another triumph of mindless mathematicism wherein superstitious-like beliefs take precedence over empirical results.

**Reply by ftessier** [reply]
Building a physical reference system on the speed of light does make more sense, yes, and that is what we do in metrology, as mentioned above.

However, by an "emergent" space, I do NOT mean derived from other units or measurements. I mean: is the notion of space itself meaningful at all in physics?

How do you know if something is near or far, for example. Try to answer this question without resorting to a priori notions of "space" or "time". Notice how "position" or "space" is a singularly human idea,  based on memorized information (known size of objects, parallax, etc.) and rather complex observations. No doubt it is useful for describing the world we observe; but is it relevant in _fundamental_ physics?

In other words, for example, how does a particle gauge location or distance? It does not, presumably. It JUST interacts. In this line of thought, I fail to grasp the physical significance of "space" or "distance" or "position (x)", etc. as an _a priori_ assumption. How can the "phenomenon" of space (or time) arise at our scale, if we assume instead that it is _not_ fundamental?

**Reply by HanDeBruijn** [reply]
>but is it relevant in _fundamental_ physics?

Okay. Herewith we proudly present: [_The Fundamental Laws of Bubble Formation_](http://www.alternatievewiskunde.nl/bellen/index.htm).
Am I serious ? Of course not ! But I want to make a point.

In the first place, @ftessier , please _define_ what **fundamental** is.
As a retired engineer, I don't know what it is. Perhaps I don't even _want_ to know what it is.
Is your "fundamental" physics so much more interesting than the physics of a humble engineer ?

Oh yes, it is !!

A quotation from Stephen Hawking: "If we do discover a [theory of everything](https://en.wikipedia.org/wiki/Theory_of_everything) [ .. ] it would be the ultimate triumph of human reason - for then we would truly know the mind of God".

Ah, here we are !! **[_The Mind of God_](https://en.wikipedia.org/wiki/The_Mind_of_God)**.

So that is what happens if people have no religion. Then they are going to make miracles of other things.
Things in nature or things made up by themselves. In the Bible, this is called **idolatry**.
The very first of the [Ten Commandments](https://en.wikipedia.org/wiki/Ten_Commandments) is devoted to exactly this: [Thou shalt have no other gods before me](https://en.wikipedia.org/wiki/Thou_shalt_have_no_other_gods_before_me).

Let us be clear about the current Mystification in physics.
Any theory that claims to be an exact description of Nature is [Built on a lie](https://www.youtube.com/watch?app=desktop&v=xaC_aKqjCXU).
As Newton has correctly formulated it - and we have forgotten that lesson: _the ART of human Technology is part of Nature_.

**Reply by Francois-Zinserling** [reply]
@ftessier I think of it this way -
 - You wouldn't know if your clock is slow. OK, if you have a previous reference to go by, you would think other events have sped up, but you wouldn't necessary know your own clock is slow if there was no change.  
 - For the concept of space it is easy to be fooled. Have you played a 3D shooter game? There all of 3D space exists in your mind and the flat screen in front of you. 

By controlling the speed of light both the above variables can be manipulated.
e.g. If your clock stops, you have no ability to make any measurements. Change (your own) clockspeed and the outside world appears to evolve at a different rate.

By launching one photon (ok, at least two photons needed) you have created a concept of time and distance, both determined by the speed of your photon(s). So both time and distance would be emergent if speed of light is fundamental, A few more from different directions and you have 3D space.

**Reply by ftessier** [reply]
@HanDeBruijn Indeed there is no exact description of Nature. There are only guesses (theory), computations and comparisons with the World. A theory cannot be proven right, it can only be proven contrary to observation. All theories are temporary, or ought to be understood as such. Historically, humans have refined their description of the World, proving theories wrong or limiting their scope (e.g., Aristotelian physics is arguably correct for objects moving in a fluid, which is what Aristotle observed, very astutely).

What I mean by a _fundamental_ theory, or should I say _more fundamental,_ is simply **one which has fewer assumptions,** especially those that betray our own biased perspective. Today we largely conceive of the World as a collection of corpuscules we call fundamental. I assume this view will be revised in the future, just as our corpuscules have replaced the ancient notion of "elements", for example. On this note, any corpuscular theory is likely ill-fated, because all we can observe are _interactions,_ not _things._ (This is a very strong ontological argument, because arguably any observation is an interaction.)

At any rate, in this line of thought, in seems to me that _a priori_ assumptions of space and time restrict modern physics to a singularly human perspective. That is what I mean: a (more) fundamental theory of physics would be one built _without_ the ingredients of space and time imposed at the outset. I trust this is possible.

**Reply by ftessier** [reply]
@Francois-Zinserling I don't agree: when an atom (which itself is already a pretty intricate system!) emits a photon (or two), it just emits the photon. No time or distance is "created" or "defined". _It just emitted a photon._ You are thinking too much like a human (!) 😉

**Reply by AndriiKyryl** [reply]
@ftessier Hello Frederic. I don't know whether you've seen my papers, but if not, you may be interested to know that a serious enough attempt to build a "truly fundamental" kind of science you're looking for has already been performed, and it does provide the kind of results you expect. And you're right, the key point is the _unreduced interaction_ analysis, instead of using heuristically guessed and in addition totally abstract ready-made structures and related "models" in the traditional approach. And _everything_ is _dynamically emerging_ there, starting from space and time. The consistent cosmological evolution is thus naturally included in this picture. The details can be found in multiple references (the full list is [here](https://sites.google.com/site/unifiedcomplexity/Andrei_Kirilyuk#h.wsoo1ovxi7ur)), you can start e.g. with https://hal.science/hal-03824740 or an older review at https://arxiv.org/abs/1204.3460. I am glad to see that you have the same general vision of the truly consistent kind of science (I call it [causally complete](https://forums.fqxi.org/d/3997-causally-complete-science-for-the-reason-based-society) science).

**Reply by ftessier** [reply]
Thank you, I will certainly consult these enthusiastically to learn more!

**Reply by Francois-Zinserling** [reply]
@ftessier you are right. I should science more and wave hands less :)
Atoms are not fundamental though. Photons are (by current understanding).

@AndriiKyryl those are some big reads! Slava!

**Reply by RedshiftDrift** [reply]
@ftessier 
> How do you know if something is near or far, for example. Try to answer this question without resorting to a priori notions of "space" or "time".

To know if an object is near or far without the notion of *space* or *time*, I build this simple device using two ballons (with opposite charges) and a needle mounted on a coil spring moved by a [rack and pinion](https://en.wikipedia.org/wiki/Rack_and_pinion).

Needle points to the **left**, the object (the balloon) is **near**; needle points to the **right**, object is **far**.
![a](https://cosmology.info/essays/marmet_l/physics-conceptC.jpg)
[From [Criteria for a Scientific Model: The measuring device as a physical concept](https://cosmology.info/essays/marmet_l/criteria-for-a-scientific-model.html)]

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
> Herewith we proudly present: [The Fundamental Laws of Bubble Formation](http://www.alternatievewiskunde.nl/bellen/index.htm).

Fun!

> ... I want to make a point. *"One cannot draw a line somewhere between the laws of physics and say: on this side everything is exact, on the other side everything is an approximation"*

  You are off the mark: The "fundamental" qualifier of 'physics' ***has nothing to do with being "exact" physics!*** There is no such thing as "exact science" even if that's what physics and other sciences are being called.

  "Fundamental" qualifies our current theories and is related to the idea of *reductionism*, where phenomena can be described in terms of other, more fundamental descriptions.

- Your 'bubble formation' law can be reduced to superficial tension, liquids, vapours, gases, etc... It is ***not*** fundamental.
- However, we don't know any 'more fundamental' laws that allows us to explain the Lorentz transformation.  This is a fundamental law.
- Newton's laws used to be fundamental.  But when Einstein developed GR, it took the place of Newton's laws as a fundamental theory of gravity.
- "Fundamental chemistry" is the study the basic structure of molecules.  It is fundamental because it cannot be understood with other laws of chemistry (but they can be understood with the laws of physics, except they are often too complex to be a tractable problem).

> is "fundamental" physics so much more interesting than the physics of a humble engineer ?

It depends what you want to do with it!
Quantum mechanics is not very interesting if your engineering goal is to build a bridge.
Quantum mechanics is very interesting if your engineering goal is to build a quantum computer.

**Reply by ftessier** [reply]
> Needle points to the left, the object (the balloon) is near; needle points to the right, object is far.

But do left and right not imply a notion of space already?

My thought is that from a corpuscule's "point of view" (fundamental physics), there is not time and space. Yet from our humble perspective, space and time are manifest to the point that it is hard to even think without them! It stands to reason that these notions arise at some stage between the two.

What I long for is a physical theory that builds "something" (anything!) out of its axioms _before_ invoking things like x and t. We always stage our theories and particles inside an x and t scaffolding at the outset (be it rigid or flexible, static or dynamic). To me that increasingly seems to be very limiting starting point. I should probably stop thinking about this 😉.

**Reply by ftessier** [reply]
@AndriiKyryl I read the references you provided (not the full list though!). Some of my own thoughts regarding modern physics echo your own, but I feel much more positively about it 🤓 Positivism (and reductionism) has made science practical and useful, explaining the World by the World. Seeking relations instead of absolutes, seeking increasing knowledge instead of _Truth._

As for your starting point, I appreciate that you aim to avoid early approximation, to keep "everything" in, so to speak. I like this program! However, I cannot bring myself to rely (in the search for a fundamental theory, that is) on assumptions here much greater than space and time: a hamiltonian dynamic, a state function, and random jumps in a space of redundant _mathematical_ solutions!

> Namely, the strongly nonlinear EP dependence on the eigenvalues η to be found leads to excessive, dynamically redundant number of problem eigen-solutions (with respect to their usually expected number), which are all equally real and describe equally possible system configurations called realisations that emerge as a result of one and the same interaction process development [1-7,17,28]. Being individually sufficient and therefore mutually incompatible, these dynamically redundant system realisations are forced, by the driving interaction itself, to permanently replace one another in a dynamically random (or chaotic) order thus rigorously defined. 

I don't understand enough of what you propose to see how, for example, space and time would in turn _emerge_ from the theory, and what the testable predictions of it might be.

**Reply by sahil5d** [reply]
There are things, and they move.

I think that's the simplest basis of reasoning.

And then that can refined to
kilogram, meter, second.

**Reply by ftessier** [reply]
Yes, exactly: space (things) and time (movement) ARE the assumptions I call into question. Are there _things?_ What is a _thing?_ The Earth? A rock? A molecule? An atom? An electron? A photon?

Consider a dual picture where there are no "things", but only events. I assume it is easier to agree that events happen, compared to admitting that _things_ exist. That is because if you divide a thing, you eventually get pieces that are different kind of _things_, whereas if you resolve events to a finer scale, all the parts are still events (i.e., they are all the same _thing!_ 🤡)

**Reply by ftessier** [reply]
@AndriiKyryl I read https://arxiv.org/pdf/physics/0410269.pdf and understand better your reasoning. I do enjoy the ideas you flesh out in there, although I would have to go through the math more intently to comment further. I like the non-perturbative, unreduced existence equation approach (funny, I once sketched out a theory I called "existence physics"!). I still have trouble though assuming a state function and a Hamiltonian from the get go.

**Reply by AndriiKyryl** [reply]
@ftessier The Hamiltonian equation form is self-consistently confirmed later (it's described in detail in the cited review papers), since it appears that totally emergent dynamics of emergent and explicitly obtained structures (starting from the elementary particles) is governed just by the same Hamiltonian equations, thus confirming the starting point. In addition, there is nothing too special in that starting "assumption" (further confirmed) as it simply states the initial system configuration and the (arbitrary) interaction fact. One could use any other suitable notation for that. But as eventually we derive the universal Hamiltonian formalism, it's more convenient to start with it and then confirm it as the universal description. The "state-function" is just a general description of the emerging system structure, i.e. greater inhomogeneity (there is no assumption here), where we further specify what is this structure and how it emerges in the unreduced interaction process. It is the role of the latter which is the key point. If you just don't introduce any restrictions (alias "models", "postulates", "exact solutions", etc.) and try to analyze the real, "nonseparable" interaction development, you obtain everything, the emerging structures (starting from particles and space they dynamically "weave" and maintain around them), real unstoppable and irreversible time, and all the formal laws and entities describing the emergent dynamics!

**Reply by ftessier** [reply]
I like self-consistency arguments, and the fact that the unreduced dynamics does reduce (if I understand correctly) to observed quantum behaviour which is, by all accounts, a fruitful description of the very small for us (humans). I shall study the math more, and probably will have many more questions!

**Reply by RedshiftDrift** [reply]
> But do left and right not imply a notion of space already?

We live in space, our experiments must therefore be "in space".
What if I paint the needle red and blue so that the colour indicates if the object is near or far?

**Reply by RedshiftDrift** [reply]
Feynman eloquently discusses "fundamental"
[https://youtu.be/oqe8ToQdacc?t=2370](https://youtu.be/oqe8ToQdacc?t=2370)

The entire video is worth watching!

**Reply by ftessier** [reply]
Indeed _we_ live in space, but particles don't. From their perspective, so to speak, there is no information about space, distance, etc. The notion of space requires a complex system (e.g., the human brain) and a web of interactions to arise. So yes, for explaining events at our scale, the approximation of empty space as a backdrop where things happen is a superb idea! But it seems to be a heavy-handed premise when trying to describe how stuff works at a  level that is _outside_ the realm of our direct perception (e.g., particle physics).

**Reply by HanDeBruijn** [reply]
Think about this.
Time is emerging from a **paradox**.
A paradox called CHANGE.
&pi;&alpha;&nu;&tau;&alpha; &rho;&epsilon;&iota; &kappa;&alpha;&iota; &omicron;&upsilon;&delta;&epsilon;&nu; &mu;&epsilon;&nu;&epsilon;&iota; ([Heraclitus](https://en.wikipedia.org/wiki/Heraclitus#Panta_rhei))
Numerical Analysis ([Fortran](https://en.wikipedia.org/wiki/Fortran)) says: $t = t + dt$ .
The integer equivalent is: $i = i + 1$ .
From which it follows that: $1 = 0$ .
[Q.E.D](https://en.wikipedia.org/wiki/Q.E.D.).

**Comment by Pierre314159** [comment]
Hi, @Han

This has been discussed a while back, and it was mathematically demonstrated that the average density of a fractally distributed infinite mass in an infinite space was indeed null. This solves both the gravitational potential paradox in Newtonian gravitation, and the need for a cosmological constant in GR. 

This is all very fine as far as mathematics are concerned. However, the problem with a fractal distribution is that physics needs “something” to keep the distribution of matter fractal. And this “something”, if you ask me, is very much alike some kind of “dark energy”.

So, here we are, back to square one.

**Reply by HanDeBruijn** [reply]
>it was mathematically demonstrated that the average density of a fractally distributed infinite mass in an infinite space was indeed null.

A _**fractally** distributed infinite mass_ is not needed in order to establish this.
With the Variable Mass Theory we find for the _average density_ $\rho$, as has been found at [another place](https://github.com/orgs/a-cosmology-group/discussions/127#discussioncomment-6826896):

$$
\frac{\rho}{\rho_0} = 3 \frac{2-e^{-x}(x^2+2x+2)}{x^3} \quad \mbox{with dimensionless} \quad x = H/c.R
$$

where $H=$ Hubble parameter, $c=$ lightspeed, $R=$ universe size. It's easy to see that, indeed

$$
\large \lim_{R\to \infty} \frac{\rho}{\rho_0}  = 0
$$

And, by the way, the mass $M_U$ of our universe is _**not** infinite_. As has been found at the same place:

$$
M_U = \frac{c^3}{w.G.H} \quad \Longrightarrow \quad 4.3 \times 10^{53} \le M_U \le 13.5 \times 10^{53} \mbox{ } kg
$$

where $1/8 \le w \le 0.388$ is a (dimensionless) number, $G=$ gravitational constant.

>This solves both the gravitational potential paradox in Newtonian gravitation, and the need for a cosmological constant in GR.

Glad to know that our VMT thus solves the same problems. Thanks !

**Comment by khuramonline** [comment]
> CP1 Physical laws the same in all space and time
> CP2 Fundamental physical constants are true constants
> CP3 Particular physical properties, including measuring standards, are the same in all space and time

CP1 depends on CP3 for fundamental particles.

CP1 is also not true if there is truth in shadow theories of gravity and it is also true that push factor comes from (practically) infinite surrounding universe.

Concept of infinite for universe.

If [sky is 100% bright](https://www.quora.com/How-can-scientists-say-that-Galaxy-Glass-z13-is-one-of-the-farthest-visible-Galaxy-when-countless-other-galaxies-are-visible-in-the-JWST-image-right-in-the-background-of-Glass-z-13/answer/Khuram-Rafique) in any form that may include 100% bright in infrared light then practically it is an infinite universe.

It takes only few thousand moons placed side by side at same radial distance to totally cover the sky.

Likewise, a finite number of galaxies with varying distances can cover whole of sky thus making the sky 100% bright. To make a 100% bright sky there is no need of an infinite universe. But practically it will be an "infinite" universe. 

CP2 ---

For example "G" is regarded as a fundamental constant. However it has been worked out on the basis of assumption that Newton Formula is m multiplied by m divided by distance square.

So what if Newton's actual law be "m + m" instead of "m x m" ...???

Newton himself has never stated m x m thing. What he actually stated is closer to m + m.

So if it is m + m, then value of G is different.

If my point about shadow gravity is correct again value of G will be different for different parts of universe.

**Reply by budrap00** [reply]
@khuramonline 

_... space-time is not necessarily something to which one can ascribe a separate existence, independently of the actual objects of physical reality. Physical objects are not in space, but these objects are spatially extended. In this way the concept "empty space" loses its meaning._
-- Albert Einstein, 1954

The sky _is_ 100% bright in microwave radiation, it _is not_ 100% bright in the infrared or any other range - by observation.

Infinity is not a meaningful physical concept and therefore it is not a meaningful scientific concept. It it suits your philosophical, metaphysical or mathematical proclivities that's fine but infinity has no place in a scientific model except as an indicator that the model is fundamentally incorrect or incomplete. 

Regards,

Bud

**Reply by ftessier** [reply]
Very good quote, of which I was not aware, thanks you! Reference?

**Reply by RedshiftDrift** [reply]
Also
>  Physical objects are not in space, but these objects are spatially extended.

The "point particle" ('point' in the mathematical sense = zero dimension) is also a simplification.  Physical objects are spatially extended.

**Reply by HanDeBruijn** [reply]
![download](https://github.com/a-cosmology-group/acg/assets/135433222/df654363-6a41-4266-ad04-7a21f1061ff9)

**Reply by budrap00** [reply]
The quote is from a Note added to the Preface for the 15th edition of _Relativity The Special And General Theory_ by Albert Einstein, describing the purpose of the newly added 5th Appendix. The 15th edition used to be freely available in digital format but a copyright was asserted and only earlier editions are still free.

**Reply by Francois-Zinserling** [reply]
Also Einstein [1920]: “Recapitulating, we may say that according to the general theory of relativity space is endowed with physical qualities; in this sense, therefore, there exists an ether. According to the general theory of relativity space without ether is unthinkable; for in such space there not only would be no propagation of light, but also no possibility of existence for standards of space and time.”

**Reply by HanDeBruijn** [reply]
@khuramonline writes:

>So what if Newton's actual law be "m + m" instead of "m x m" ...???

It must be **multiplication**, without doubt. But you are not the only one who has problems with the multiplication of physical quantities. Here is a quote from one of my favorite books: [Origin of Inertia](http://www.alternatievewiskunde.nl/cosmology/ACG_refs/96836893X-rev.pdf), found at page 139, written by [Amitabha Ghosh](https://en.wikipedia.org/wiki/Amitabha_Ghosh_(academic,_born_1941)). And he is not a little boy.

>What was very difficult for me at that time was to conceive of the idea of one physical
quantity being multiplied by another physical quantity. For me the stumbling block
was the concept of momentum: the product of mass and velocity. I can still remember
the utter exasperation of the young law student who had already completed a Master of
Science in Mathematics from Calcutta University. He was completely baffled by my
difficulty. It took a long time for me to accept the concept of momentum.

And here is a Question &amp; Answer @ Mathematics Stack Exchange about the same: [Is this proportionality proof erroneous?](https://math.stackexchange.com/questions/1723509/is-this-proportionality-proof-erroneous)

**Reply by budrap00** [reply]
@Francois-Zinserling That 1920 quote is from an address Einstein gave at the University of Leyden. There is little in the historical record to suggest that Einstein retained that view regarding the aether beyond that early time frame. Einstein was, like everyone else, still thinking his way through all the implications of GR in 1920.

It is interesting to compare the 1920 statement, "_According to the general theory of relativity space without ether is unthinkable_", with this quote from the 5th Appendix (referenced above) from 1954 (1952 in the original German), "_Space-time does not claim existence on its own, but only as a structural quality of the field._" It would appear that Einstein's thinking evolved from the vague notion of an aether to the more concrete concept of a field with structural qualities.

It is also interesting to consider that Einstein was a colleague of John A. Wheeler at Princeton at the time the 5th Appendix was written. In that context it is hard not to consider the 5th A. a direct refutation of Wheeler's concept of a causally interacting space-time (that tells matter how to move, etc.)

**Reply by RedshiftDrift** [reply]
>> What was very difficult for me at that time was to conceive of the idea of one physical quantity being multiplied by another physical quantity. For me the stumbling block was the concept of momentum: the product of mass and velocity.

But ***velocity*** is a *ratio* of the different physical quantities $m$ and $s$ - that didn't bother him?
The Taj Mahal was built from approx. $10^8 \ person \cdot hour$ of ***work***, is that also difficult to conceive?
When Ghosh returned to his 15 $m^2$ abode that evening, was the concept of ***area*** a stumbling block too?

Einstein was right...
